src/features/bookings/pages/EndedBookings/EndedBookings.perf.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { rest } from 'msw'
+import React from 'react'
+
+import { bookingsSnap } from 'features/bookings/fixtures/bookingsSnap'
+import { EndedBookings } from 'features/bookings/pages/EndedBookings/EndedBookings'
+import { beneficiaryUser } from 'fixtures/user'
+import { env } from 'libs/environment'
+import { reactQueryProviderHOC } from 'tests/reactQueryProviderHOC'
+import { server } from 'tests/server'
+import { measurePerformance, screen } from 'tests/utils'
+
+server.use(
+  rest.get(env.API_BASE_URL + '/native/v1/me', (_req, res, ctx) =>
+    res(ctx.status(200), ctx.json(beneficiaryUser))
+  )
+)
+
+server.use(
+  rest.get(`${env.RECOMMENDATION_ENDPOINT}/native/v1/bookings`, (_req, res, ctx) =>
+    res(ctx.status(200), ctx.json(bookingsSnap))
+  )
+)
+
+describe('<EndedBookings />', () => {
+  it('Performance test for EndedBookings page', async () => {
+    // eslint-disable-next-line local-rules/no-react-query-provider-hoc
+    await measurePerformance(reactQueryProviderHOC(<EndedBookings />), {
+      // Add scenario if necessary
+      scenario: async () => {
+        await screen.findByText('Réservations terminées')
```

Comment:
Je pense qu'on n'attend l'affichage du mauvais élément (on attend l'affichage du header ici, qui lui est toujours affiché, alors qu'on devrait attendre l'affichage d'une réservation).
```suggestion
        await screen.findByText('bookingsSnap.ended_bookings[0].stock.offer.name')
```

Pour que la route `/native/v1/bookings` soit appelé, il faut que l'utilisateur soit connecté, tu peux mocker ça grâce à `AuthContext` :
```
jest.mock('features/auth/context/AuthContext', () => ({
  useAuthContext: jest.fn(() => ({ isLoggedIn: true })),
}))
```